src/components/tables/DataTable/table/DataTable.tsx

```diff
@@ -241,9 +241,14 @@ export const DataTable = <D extends object>(props: DataTableProps<D>) => {
                 ]);
                 
                 const useExtraColSpan = requireNewCol(column);
-                
+                const getAriaSort = () => {
+                  if (!column.canSort || !column.isSorted) return undefined;
```

Comment:
Can you wrap the `return` in braces? Just to prevent accidental bugs if in the future another statement gets added to the `if` statement.